commit 41293b8321939958ae1664e41b02380440d2ceab
Author: Rafael J. Wysocki <rjw@sisk.pl>
Date:   Wed Dec 15 00:17:29 2010 +0100

    PM: Avoid compiler warning in pm_noirq_op()
    
    The compiler complains that calltime may be uninitialized in
    pm_noirq_op(), so add extra initialization for that variable to
    avoid the warning.
    
    Signed-off-by: Rafael J. Wysocki <rjw@sisk.pl>

diff --git a/drivers/base/power/main.c b/drivers/base/power/main.c
index 4de226a..a51a04f 100644
--- a/drivers/base/power/main.c
+++ b/drivers/base/power/main.c
@@ -309,7 +309,7 @@ static int pm_noirq_op(struct device *dev,
 			pm_message_t state)
 {
 	int error = 0;
-	ktime_t calltime, delta, rettime;
+	ktime_t calltime = ktime_set(0, 0), delta, rettime;
 
 	if (initcall_debug) {
 		pr_info("calling  %s+ @ %i, parent: %s\n",
